client/src/components/App/Content/SignUp/SignUpForm/useSignUpForm.ts

```diff
@@ -62,22 +63,58 @@ const useSignUp = ({ handleSaveUser }: useSignUpFormInCome) : useSignUpFormOutCo
                 alertError('לא ניתן היה לקבל שפות');
                 fetchLanguagesLogger.error('didnt get results back from the server', Severity.HIGH);    
             });
-    }
+    };
+    
+    const fetchDesks = (countyId?: number) => {            
+        const fetchDesksLogger = logger.setup('Getting desks');
+        if (!countyId) {
```

Comment:
maybe you can change the order of the if/else or export to const for avoiding the negative condition